app/models/concerns/eventable.rb

```diff
@@ -4,12 +4,16 @@ module Eventable
   included do
     has_many :events, as: :eventable, dependent: :destroy
 
-    after_create { record(:created) }
+    before_create { record(:created) }
   end
 
-  private
```

Comment:
Should we keep `record` private and just use `send` in the script? Normally I would advocate against this, but it seems like more of an edge case.

Reply:
yeah that's a good idea